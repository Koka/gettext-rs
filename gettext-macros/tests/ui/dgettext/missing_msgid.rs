use gettext_macros::*;

fn main() {
    dgettext!("domainname");
    dgettext!("domainname", 1, 2);
}
