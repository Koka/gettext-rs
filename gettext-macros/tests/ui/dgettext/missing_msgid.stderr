error: missing msgid string argument
 --> tests/ui/dgettext/missing_msgid.rs:4:5
  |
4 |     dgettext!("domainname");
  |     ^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `dgettext` (in Nightly builds, run with -Z macro-backtrace for more info)

error: missing msgid string argument
 --> tests/ui/dgettext/missing_msgid.rs:5:29
  |
5 |     dgettext!("domainname", 1, 2);
  |                             ^
